### DISCUSSION TITLE
Best way to deploy if I have lots of data

### DISCUSSION THREAD

**Discussion by zanzlender** [nextapps-de/flexsearch#388] [General]
I have just started a project recently where I wanted to try out FlexSearch. And soon I found myself in a problem where I have a lot of indexes (like +1million).

And I was wondering what would be the best way to deploy this so that I can also have my data stored inside a JSON or some similar file. Because running import for this many indexes is kinda slow... And as far as I've read in other discussions FlexSeach Indexes are not in-memory. Would that mean the flexSearch is not ideal for that many indexes?

If anyone has an idea or wants to share how they do it I would appreciate it :)

**Comment by howesteve** [comment]
If you have that much documents to index, obviously you should index them on a server, using a server solution, not on the browser. This project is for much smaller document sets.
You are probably looking for elasticserach, meilisearch, zincsearch, algolia, etc.
